src/app/admin/questions/page.tsx

```
@@ -1,582 +1,382 @@
+									<label htmlFor="text" className="text-sm font-medium">Question Text</label>
+									<textarea
+										id="text"
+										value={newQuestion.text}
+										onChange={(e) => setNewQuestion({ ...newQuestion, text: e.target.value })}
+										className="flex min-h-[100px] w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50"
+										placeholder="Enter your question here..."
+									/>
+								</div>
+								<div className="grid grid-cols-2 gap-4">
+									<div className="grid gap-2">
+										<label className="text-sm font-medium">Style</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)